### ISSUE TITLE
[BUG] Serialize option for version incorrect data type in source

### ISSUE THREAD

**Issue by kirill-mikhaylov** [mathematic-inc/ts-japi#15]
## Describe the bug<sup>*</sup>

In the documentation for `Serializer` states to set `version` option to `null` to disable `jsonapi` version output. It does not work for `ErrorSerializer`. Currently it has only 2 allowed data types: `string | undefined`. Please add `null` to replicate the same functionality as in `Serializer`.

## To Reproduce<sup>*</sup>

```
const errorResponse = new ErrorSerializer();
const newError = new JapiError({
    detail: 'Short concise description of the error',
    status: '400'
});
console.log(errorResponse.serialize(newError, { version: null }));
```

## Expected behavior<sup>*</sup>

Setting `null` on `version` option for `ErrorSerializer` to behave the same as documented for `Serializer`.

## Screenshots

![Untitled-1](https://user-images.githubusercontent.com/320758/89162219-972df100-d528-11ea-850b-a0c2012edf10.jpg)

## Platform<sup>*</sup>

- [x] OS: Win10
- [x] Browser Google Chrome Version 84.0.4147.105 (Official Build) (64-bit)
- [x] Version: `"ts-japi": "^1.3.0"`

## Additional context

`version` documentation for Serializer: https://mu-io.github.io/ts-japi/interfaces/serializeroptions.html#version
`version` documentation for ErrorSerializer: https://mu-io.github.io/ts-japi/interfaces/errorserializeroptions.html#version

\* means required.

**Comment by jrandolf** [comment]
What happens if you set it anyway? Using "as any"

**Comment by kirill-mikhaylov** [comment]
Good suggestion, however, I would personally prefer to omit hacks like that and rather make use of explicit definition. The suggestion works producing the following response object:

![Untitled-1](https://user-images.githubusercontent.com/320758/89254815-f0ebf500-d5d4-11ea-8588-4df19ee01868.png)

This interim solution is good to force transpiler to not complain, but evades the whole functional purpose of setting it to `null` to hide the `version` from response object. Of course I can add my own filter to remove the `version` before sending it back to client, but I favor your library and believe the consistency in experience is key.

**Comment by jrandolf** [comment]
> This interim solution is good

Actually, by doing the proposed workaround, I was hoping it would actually just work, but I saw where the issue was.

In any case, the fix has been published.

**Comment by kirill-mikhaylov** [comment]
My hats off to you sir - thank you for being so prompt.

**Comment by kirill-mikhaylov** [comment]
Pulled, tested, and validating the fix (`1.3.1`) - `null` is now being recognized on `version` and setting it omits the `version` from the response object. I would also suggest to update description on `ErrorSerializer` `version` docs to match `Serializer` by appending:
```
Set to null to omit version.
```
in https://mu-io.github.io/ts-japi/interfaces/errorserializeroptions.html#version
